modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/feature/VulcanFeature.java

```diff
@@ -174,6 +175,8 @@ _roleLocalService, _getScopeChecker(), _sortParserProvider,
 		featureContext.register(
 			new SiteParamConverterProvider(
 				_depotEntryLocalService, _groupLocalService));
+		featureContext.register(
```

Comment:
By including this ConverterProvider in VulcanFeature, validations will be performed for all the APIs created by REST Builder, not only Object's one.

Vulcan does not know about Objects

Reply:
Do we have any other use of scopeKey in rest layer?
Looking into the code, I just see this as object related.